### PR TITLE
add health / refresh dependency

### PR DESCRIPTION
**Pull request by luckylhb90**

### CODE REVIEW

File: internal/controller/router.go [hunk 1]
```diff
@@ -4,24 +4,43 @@ import (
...
-		GetDependencies,
-	)
+	PrivateGroup.Use(middleware.Auth())
 
-	eng.POST(
-		"/v1/sandbox/dependencies/update",
-		UpdateDependencies,
-	)
+	{
+		// 健康监测
```
Comment (Yeuoly) [inline internal/controller/router.go:17]: pls use English instead.

**Review by Yeuoly — CHANGES_REQUESTED** [review]

File: internal/controller/router.go (hunk 1 above)
Comment (luckylhb90) [reply internal/controller/router.go]: fixed

**Comment by luckylhb90** [comment]
solved #22

**Review by Yeuoly — APPROVED** [review]
LGTM

**Pull request merged**